docs/proposals/horizontal-pod-autoscaler.md

```diff
@@ -200,16 +200,19 @@ and adjust the count of the Scale if needed to match the target
 The target number of pods will be calculated from the following formula:
 
 ```
-TargetNumOfPods = sum(CurrentPodsConsumption) / Target
+TargetNumOfPods = 1 + floor(sum(CurrentPodsConsumption) / Target)
```

Comment:
OK. It's the same. 
I added an extra condition which prevents scaling to 0 (will be removed once we have idleing).